docs/spec/modules.md

```diff
@@ -1,53 +1,57 @@
-# Module
-> **Note**: Module is not implemented yet. 
-
-## Definition
+# Modules
 
 A module is an opaque set of one or more resources to be deployed together. It only exposes parameters and outputs as contract to other bicep files, hiding details on how internal resources are defined. This allows you to abstract away complex details of the raw resource declaration from the end user who now only needs to be concerned about the module contract. Parameters and outputs are optional.
 
-## Declare a module
-
-Any bicep file is itself a module, so there is no specific syntax for defining a module. A module can be a single file or a directory. If a module references a directory, all root files in that directory will be combined. Here is an example bicep file (`sqlDatabases.bicep`) that we will consume as a module:
```

Comment:
Should change "Any bicep file" to "Every bicep file" to make a stronger statement.